### ISSUE TITLE
updated docs for contributing

### ISSUE THREAD

**Issue by jvolkening** [bioconda/bioconda-recipes#7375]
The [official docs](https://bioconda.github.io/contribute-a-recipe.html) on testing new recipes reference `simulate-travis.py` but that script seems to have disappeared from `master`, presumably due to the recent switch to Circle CI.

Is there a new recommended method for testing new recipes in a clean environment prior to submitting a PR?

**Comment by johanneskoester** [comment]
Yes, you can install the [Circle CI client](https://circleci.com/docs/2.0/local-jobs/#installing-the-cli-locally), and simply run `circleci build`. Note that this needs a working docker installation with your user in the docker group. We might restore the old simulate script but in the long run, this will become the default and the docs will be updated once we are sure that circleci is the proper solution to all our challenges.

**Comment by nathanweeks** [comment]
I'm getting the following error regarding the "git" command missing when I attempt to use `circleci build`:

```
bash-4.2$ type git
git is hashed (/usr/bin/git)
bash-4.2$ circleci build

====>> Spin up Environment
Build-agent version 0.0.4666-05dad47 (2018-01-25T18:14:33+0000)
Starting container bioconda/bioconda-utils-build-env
  using image bioconda/bioconda-utils-build-env@sha256:11ef017d5a6fd3fd6b06b1e6f6a8607b5ca42e352107a9c462516560eaf5ac3d

Using build environment variables:
  BASH_ENV=/tmp/.bash_env-localbuild-1517002172
  CI=true
  CIRCLECI=true
  CIRCLE_BRANCH=augustus-3.3
  CIRCLE_BUILD_NUM=
  CIRCLE_JOB=build
  CIRCLE_NODE_INDEX=0
  CIRCLE_NODE_TOTAL=1
  CIRCLE_REPOSITORY_URL=git@github.com:nathanweeks/bioconda-recipes.git
  CIRCLE_SHA1=d26a1349fb27c2e7c4788f0b00c7c76846b96a59
  CIRCLE_SHELL_ENV=/tmp/.bash_env-localbuild-1517002172
  CIRCLE_WORKING_DIRECTORY=~/project

====>> Checkout code
  #!/bin/sh
mkdir -p /root/project && cp -r /tmp/_circleci_local_build_repo/. /root/project
====>> Setup ssh
  #!/bin/bash -eo pipefail
mkdir -p ~/.ssh
ssh-keyscan -H github.com >> ~/.ssh/known_hosts

# github.com SSH-2.0-libssh_0.7.0
====>> Download common definitions
  #!/bin/bash -eo pipefail
curl -s https://raw.githubusercontent.com/bioconda/bioconda-utils/master/.circleci/common.sh > .circleci/common.sh
====>> Setup bioconda-utils
  #!/bin/bash -eo pipefail
.circleci/setup.sh
.circleci/setup.sh: line 14: type: bioconda-utils: not found
Setting up bioconda-utils...
... edited for brevity ...
conda-build-2. 100% || Time: 0:00:00   7.80 MB/ss
Collecting git+https://github.com/bioconda/bioconda-utils.git@v0.11.3
  Cloning https://github.com/bioconda/bioconda-utils.git (to v0.11.3) to /tmp/pip-xmjqkm6m-build
  Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/bioconda/bioconda-utils.git /tmp/pip-xmjqkm6m-build
Cannot find command 'git'
Error: Exited with code 1
Step failed
Task failed
```

**Comment by johanneskoester** [comment]
I'll have a look on monday. Thanks for reporting! Could it be that you had the bioconda-utils-build-env container before? It seems that it has not been updated by the circleci client, and those missing binaries were only added yesterday.

**Comment by johanneskoester** [comment]
Please try a `docker pull bioconda/bioconda-utils-build-env` first.

**Comment by nathanweeks** [comment]
@johanneskoester : that did the trick. Thanks!

**Comment by kpepper** [comment]
I just got the same 'Cannot find command git' issue, so tried pulling the bioconda/bioconda-utils-build-env container but circleci is still failing with the same error.

**Comment by johanneskoester** [comment]
Should be fixed with PR #7509. Feel free to reopen if it still does not work.
